Dockerfile

```diff
@@ -21,7 +21,20 @@ RUN apt-get install -y \
     xfonts-base \
     xfonts-encodings \
     xfonts-utils \
-	&& wget https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.buster_amd64.deb \
+    # Install libssl1.1_1 (required for wkhtmltopdf)
+    && wget -q http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_amd64.deb \
+    && dpkg -i libssl1.1_1.1.1f-1ubuntu2_amd64.deb \
+    # Install wkhtmltopdf
+	&& wget -q https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.buster_amd64.deb \
 	&& dpkg -i wkhtmltox_0.12.5-1.buster_amd64.deb \
-	&& apt --fix-broken install
-RUN rm -rf /var/lib/apt/lists/*
+	&& apt --fix-broken install \
+    # Cleanup
+    && rm *.deb \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Cleanup should only happen at the very end.

Reply:
AFAIK doing it only at the end will not make the image smaller because the lists are included in the previous layers. See https://stackoverflow.com/a/61990740/9478384

Reply:
Makes sense, you're right!